develop/dev-guide-gui-dbeaver.md

```
@@ -132,13 +132,13 @@ TiDBはMySQL互換データベースであり、 [DBeaverコミュニティ](htt
 
 2.  右上隅の**「接続」**をクリックしてください。接続ダイアログが表示されます。
 
-3.  接続ダイアログで、「**接続タイプ」**ドロップダウンリストから**「パブリック」**を選択し、 **「CA証明書」**をクリックしてCA証明書をダウンロードします。
+3.  接続ダイアログで、**接続タイプ**ドロップダウンリストから**「パブリック」**を選択し、 **「CA証明書」**をクリックしてCA証明書をダウンロードします。
```

markdownlint-cli2 (0.23.2)

[warning] Spaces after list markers
Expected: 1; Actual: 2

(MD030, list-marker-space)